src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/InputJavadocPackageBadCls.java

```diff
@@ -1,4 +1,5 @@
-package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage;      // violation
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage;
+// violation above 'Missing package-info.java file'
 
 /* Config:
  * allowLegacy = "false"
```

Comment:
move config comment to most top.
update https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java to use method like
https://github.com/checkstyle/checkstyle/blob/423c0a797a3c2eb00493c72e45f3acc4159ec2cb/src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AtclauseOrderCheckTest.java#L60

Reply:
Config has been moved to the top @romani .  Methods are being changed to verifyWithInlineConfigParser().

Reply:
@romani config message has been placed at the top appropriately please check. Also do I need to change the method to ``` verfiyWithInlineConfigParser``` as it is not a part of the issue https://github.com/checkstyle/checkstyle/issues/11214

Reply:
Yes, we need new method to be used